src/core/shards/core.hpp

```diff
@@ -1989,7 +2026,15 @@ struct Clear : SeqUser {
 };
 
 struct Drop : SeqUser {
-  static SHTypesInfo inputTypes() { return CoreInfo::AnyType; }
+  static SHOptionalString help() {
+    return SHCCSTR("Drops the last element of the sequence variable passed in the `:Name` parameter. Works only on sequences.");
+  }
+
+  static SHTypesInfo inputTypes() { return CoreInfo::NoneType; }
```

Comment:
Why `None`?

Reply:
Reverted this change (rebased and force-pushed to this commit).